packages/flutter/lib/src/material/carousel.dart

```diff
@@ -107,6 +107,15 @@ import 'theme.dart';
 /// Here is an example to show different carousel layouts that [CarouselView]
 /// and [CarouselView.weighted] can build.
 ///
+/// On desktop and web running on desktop platforms, dragging to scroll with a mouse is disabled by default
+/// to align with the natural behavior found in other desktop applications.
+///
+/// Nevertheless, it remains feasible to scroll horizontally using a mouse by pressing Shift key while scrolling with the mouse wheel.
+///
+/// This behavior is dictated by the [ScrollBehavior], and can be changed by adding
+/// [PointerDeviceKind.mouse] to [ScrollBehavior.dragDevices].
+/// For more info on this, please refer to https://docs.flutter.dev/release/breaking-changes/default-scroll-behavior-drag
```

Comment:
```suggestion
/// On desktop and web running on desktop platforms, dragging to scroll with a mouse is disabled by default
/// to align with natural behavior.
///
/// To further align expected behavior like this, mouse input can scroll horizontally by pressing  the shift key while scrolling with the mouse wheel.
///
/// This key-driven behavior is dictated by the [ScrollBehavior.pointerAxisModifers], 
/// while [ScrollBehavior.dragDevices] manages what devices can drag a scrollable.
```

We don't need to refer to old breaking changes docs. They are not relevant to new users. :)

Reply:
Thanks for reviewing this, Kate! The change looks much more seamless and intuitive.